app/services/gtfs_graph.rb

```diff
@@ -15,38 +17,44 @@ def initialize(filename, feed, feed_version)
     @onestop_id_to_entity = {}
   end
 
-  def create_change_osr(import_level=0)
-    raise ArgumentError.new('import_level must be 0, 1, or 2.') unless (0..2).include?(import_level)
-    log "Load GTFS"
+  def create_change_osr(import_level = 0)
+    fail ArgumentError, 'import_level must be 0, 1, or 2.' unless (0..2).include?(import_level)
+    log 'Load GTFS'
     @gtfs.load_graph
-    log "Load TL"
+    log 'Load TL'
     load_tl_stops
     load_tl_routes
     operators = load_tl_operators
-    routes = operators.map { |operator| operator.serves }.reduce(Set.new, :+)
-    stops = routes.map { |route| route.serves }.reduce(Set.new, :+)
-    log "Create changeset"
-    changeset = Changeset.create()
-    log "Create: Operators, Stops, Routes"
+    routes = operators
+             .map(&:serves)
+             .reduce(Set.new, :+)
+             .map { |i| find_by_onestop_id(i) }
+    stops = routes
+            .map(&:serves)
+            .reduce(Set.new, :+)
+            .map { |i| find_by_onestop_id(i) }
+    operators.each { |o| o.serves = nil }
+    log 'Create changeset'
+    changeset = Changeset.create!
+    log 'Create: Operators, Stops, Routes'
     # Update Feed Bounding Box
-    log "  updating feed bounding box"
+    log '  updating feed bounding box'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.